recipes/mauvealigner/meta.yaml

```diff
@@ -38,3 +38,4 @@ about:
   license_family: GPL
   summary: The mauveAligner and progressiveMauve command-line tools for generating multiple genome alignments in the presence of large-scale evolutionary events
   dev_url: https://sourceforge.net/projects/mauve/
+
```

Comment:
You can also create empty commits using the `--allow-empty` argument, e.g., via
```
git commit --allow-empty -m 'trigger build'
```

Reply:
Neat...every day's a school day...